docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md

```diff
@@ -0,0 +1,361 @@
+d6b3de42c623        10.8.101.2:5000/rancher/rke-tools:v0.1.66             "/docker-entrypoint.…"   6 minutes ago       Up 6 minutes                            etcd-rolling-snapshots
+923998518d49        10.8.101.2:5000/rancher/coreos-etcd:v3.4.3-rancher1   "/usr/local/bin/etcd…"   6 minutes ago       Up 6 minutes                            etcd
+```
+```
+NAME              STATUS   ROLES               AGE   VERSION
+node/10.50.2.10   Ready    controlplane,etcd   68s   v1.18.12
+node/10.50.2.11   Ready    controlplane,etcd   68s   v1.18.12
+node/10.50.2.12   Ready    controlplane,etcd   68s   v1.18.12
+node/10.50.2.20   Ready    worker              67s   v1.18.12
+node/10.50.2.21   Ready    worker              67s   v1.18.12
+node/10.50.2.22   Ready    worker              67s   v1.18.12
+
+NAMESPACE       NAME                                           READY   STATUS      RESTARTS   AGE
+ingress-nginx   pod/default-http-backend-5b564dd459-stjgz      1/1     Running     0          32s
+ingress-nginx   pod/nginx-ingress-controller-blz4j             1/1     Running     0          32s
+ingress-nginx   pod/nginx-ingress-controller-cj8nd             1/1     Running     0          32s
+ingress-nginx   pod/nginx-ingress-controller-fqjdh             1/1     Running     0          32s
+kube-system     pod/calico-kube-controllers-6c6fc476f6-82wzb   1/1     Running     0          48s
+kube-system     pod/calico-node-7dz2m                          1/1     Running     0          47s
+kube-system     pod/calico-node-cm5rc                          0/1     Running     0          47s
+kube-system     pod/calico-node-dmwbz                          0/1     Running     0          47s
+kube-system     pod/calico-node-kf6zb                          0/1     Running     0          47s
+kube-system     pod/calico-node-nczbb                          0/1     Running     0          47s
+kube-system     pod/calico-node-zjtmj                          1/1     Running     0          47s
+kube-system     pod/coredns-5dd4dfcb45-lvlc4                   1/1     Running     0          10s
+kube-system     pod/coredns-5dd4dfcb45-sj44t                   1/1     Running     0          44s
+kube-system     pod/coredns-autoscaler-557f965569-4sbdd        1/1     Running     0          43s
+kube-system     pod/metrics-server-77956db857-pdklq            1/1     Running     0          38s
+kube-system     pod/rke-coredns-addon-deploy-job-2h25s         0/1     Completed   0          46s
+kube-system     pod/rke-ingress-controller-deploy-job-9882t    0/1     Completed   0          36s
+kube-system     pod/rke-metrics-addon-deploy-job-l47wt         0/1     Completed   0          41s
+kube-system     pod/rke-network-plugin-deploy-job-xzrwp        0/1     Completed   0          56s
+```
+
+### 3.2 OFFLINE / AIR-GAPPED
+
+- not fully supported / automated (but easy to implement)
```

Comment:
can you please provide links to resources explaining those two things?

Reply:
added definitions what "offline mode" and "airgapped mode" mean in this document and also some links to documentation